### PR TITLE
backport: trivial 2024 02 12

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## Issue being fixed or feature implemented
Batch of trivial backports

## How Has This Been Tested?
Built; ran functional tests

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: src/qt/intro.cpp [hunk 1]
```diff
@@ -367,6 +387,11 @@ void Intro::UpdatePruneLabels(bool prune_checked)
         storageRequiresMsg = tr("Approximately %1 GB of data will be stored in this directory.");
     }
     ui->lblExplanation3->setVisible(prune_checked);
+    ui->pruneGB->setEnabled(prune_checked);
+    static constexpr uint64_t nPowTargetSpacing = 10 * 60;  // from chainparams, which we don't have at this stage
```
Comment (UdjinM6) [inline src/qt/intro.cpp:391]: gui-125:
```suggestion
    static constexpr uint64_t nPowTargetSpacing = 2.5 * 60;  // from chainparams, which we don't have at this stage
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Review by UdjinM6 — DISMISSED** [review]
utACK

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v20.1.0-devpr5870.86d8aeb2. A new comment will be made when the image is pushed.

**Review by knst — CHANGES_REQUESTED** [review]

File: src/qt/intro.cpp [hunk 2]
```diff
@@ -367,6 +387,11 @@ void Intro::UpdatePruneLabels(bool prune_checked)
         storageRequiresMsg = tr("Approximately %1 GB of data will be stored in this directory.");
     }
     ui->lblExplanation3->setVisible(prune_checked);
+    ui->pruneGB->setEnabled(prune_checked);
+    static constexpr uint64_t nPowTargetSpacing = 2.5 * 60;  // from chainparams, which we don't have at this stage
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/qt/intro.cpp:391]: meh; it's done at compile time, I think we can trust the compiler here

File: doc/build-openbsd.md [hunk 3]
```diff
@@ -61,22 +61,29 @@ export AUTOMAKE_VERSION=1.15 # replace this with the automake version that you i
...
+hence you have to explicitely disable it by passing the parameter
+`--disable-external-signer` to the configure script.
+(Background: the feature requires the header-only library boost::process, which
+is available on OpenBSD 6.9 via Boost 1.72.0, but contains certain system calls
+and preprocessor defines like `waitid()` and `WEXITED` that are not available.)
+
 To configure with wallet:
 ```bash
-./configure --with-gui=no CC=cc CXX=c++ \
+./configure --with-gui=no --disable-external-signer CC=cc CXX=c++ \
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply doc/build-openbsd.md]: done

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v20.1.0-devpr5870.f57f5d66. A new comment will be made when the image is pushed.

File: src/qt/intro.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/qt/intro.cpp:391]: Also; this is exactly how it's set in chainparams; so it least if there's a compiler bug we will be consistent. :D

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v20.1.0-devpr5870.4c6f8d3b. A new comment will be made when the image is pushed.

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has began to build this PR tagged as v20.1.0-devpr5870.8bb2beb4. A new comment will be made when the image is pushed.

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has completed; a release should be present here: https://github.com/dashpay/dash-dev-branches/releases/tag/v20.1.0-devpr5870.8bb2beb4. The image should be on dockerhub soon.

**Comment by DashCoreAutoGuix** [comment]
Guix Automation has completed; a release should be present here: https://github.com/dashpay/dash-dev-branches/releases/tag/v20.1.0-devpr5870.bf3d2d44. The image should be on dockerhub soon.

**Review by UdjinM6 — APPROVED** [review]
re-utACK, CI failure https://gitlab.com/dashpay/dash/-/jobs/6173511888 looks unrelated (restarted it just in case)

**Review by knst — APPROVED** [review]
LGTM

**Pull request merged**